Dynatrace/dynatrace-operator
# Create edgeconnect.yaml config file and use it inside edgeConnect deployment

## Description

This PR creates mechanism to configure EdgeConnect via `edgeConnect.yaml` as a secret. Also `edgeConnect.yaml` is mounted to the actual container and is used.

No more Env Vars are used to configure EdgeConnect.

Secrets OAuthClientID & OAuthClient Secret are also passed via `edgeConnect.yaml`.

`caCertsRef` is mounted to a specific path of the container and this path is correctly specified in edgeConnect.yaml

## How can this be tested?

- Create regular EdgeCOnnect custom resource, make sure it's up and running.
- Create provised=true EdgeConnect, make sure it's up and running.


## Checklist
